docs/program/apis/robot.md

```diff
@@ -0,0 +1,410 @@
+---
+title: "Manage Robots with Viam's Client SDKs"
+linkTitle: "Robot Management"
+weight: 20
+type: "docs"
+description: "How to use the Robot API to monitor and manage your robots."
+tags: ["robot state", "sdk", "apis", "robot api"]
+---
+
+The *robot API* is the designated interface for a robot, the root of all robotic parts.
```

Comment:
```suggestion
The *robot API* is the API that manages each of your robots running `viam-server`.
It is Viam's designated application programing interface for a robot, the root of all robotic parts.
```
I think you didn't write this phrasing initially but would suggest updating to this or similar for easy comprehension by users of what an API is and that there is one created for each robot, individually, when you start it up (following the same set of Robot API instructions to instantiate, but individual instances). Eric's original phrasing is simple and easy for someone like IT Eric to understand but I think more front matter here would be better

Reply:
can we include the glossary entry for part?

Reply:
Indeed I was just moving content onto a separate page but I agree that it doesn't make sense to less technical users. However, we don't usually spell out API on each page. If you think we should, how about in the first sentence:
```suggestion
The *robot API* is the application programming interface that manages each of your robots running `viam-server`.
It is the API for high level operations of each robot {{< glossary_tooltip term_id="part" text="part" >}}.
```
Also I'm not sure that "root" will make sense to newer folks either when used in this way.

Reply:
yes let's make the change you suggested as a fly-by fix